src/app/shared/services/auth.service.ts

```diff
@@ -0,0 +1,87 @@
+import { Injectable, NgZone } from '@angular/core';
+import { AngularFireAuth } from '@angular/fire/compat/auth';
+import { AngularFirestore } from '@angular/fire/compat/firestore';
+import { Router } from '@angular/router';
+
+import { USER } from '@constants/local-storage';
+import { ROUTER } from '@constants/router';
+
+import { LocalStorage } from '@services/local-storage';
+
+import { User } from '@typings/user';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class AuthService {
+  userData?: User; // Save logged in user data
```

Comment:
remove such comments or replace with https://tsdoc.org/

Reply:
if you want left comments - leave it in all components, services etc.
you can do this when finish all logic

Reply:
Sorry. Got this comment from tutorial. Will remove.

Reply:
Fixed